cmd/admin_test.go

```diff
@@ -270,7 +270,7 @@ var _ = Describe("Namespace integration tests", func() {
 
 			tempSettings.AppendPostHandler(token, clitest.MockRequestResponse{
 				Status:   http.StatusOK,
-				Request:  expectedDeleteNamespacerequest,
+				Request:  expectedDeleteNamespaceRequest,
 				Response: gqlDeleteNamespaceResponse})
```

Comment:
tries to keep the camelCase consistent, as you can see with `gqlDeleteNamespaceResponse`